modules/migration-debugging-velero-resources.adoc

```diff
@@ -42,6 +42,8 @@ $ oc -n {namespace} exec deployment/velero -c velero -- ./velero \
   --help
 ----
 
+// Wes this is the describe section. What changes do I need to make
```

Comment:
Wes this is the `describe` module in the Troubleshooting Guide

What do I need to change?

Reply:
Did you want to remove these?